Jenkinsfile

```diff
@@ -75,9 +75,14 @@ pipeline {
       steps {
         timeout(90) {
           sh "make buildbottest 'TESTOPTS=-j2 --junit-xml test-results.xml -j4 \${BUILDBOT_TESTOPTS}' TESTPYTHONOPTS="
-          junit 'test-result.xml'
         }
       }
     }
   }
+
+  post {
+    always {
+      junit 'test-result.xml'
```

Comment:
```suggestion
      junit 'test-results.xml'
```

Reply:
ah, Thanks for catching the my silly typo